cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
The `include` function is using the `namespace` argument, which requires the included URLconf to have an `app_name` attribute defined. Ensure that the `cinema.urls` and `user.urls` modules have an `app_name` attribute set to avoid runtime errors.